examples/README.md

```diff
@@ -98,6 +99,15 @@ These user could customize and run these jobs over OpenPAI.
   1. [Open MPI TensorFlow CIFAR-10](./mpi#open-mpi-tensorflow-cifar-10-example)
   2. [Open MPI CNTK grapheme-to-phoneme conversion](./mpi#open-mpi-cntk-grapheme-to-phoneme-conversion-example)
 
+## What if the example is failed <a name="debug"></a>
+
+The example in the folder could be failed due to the following reasons:
+
+  1. The key in json file is removed. You may get error when you copy the json file to the webportal. You should refer to the latest version of webportal.
+  2. The docker image is removed. You will find this error in your job tracking page. Rebuild the image and push it to solve this problem.
```

Comment:
I mean "If the webportal delete some keys, the format of the old json files of these examples will incorrect".

Reply:
fix

Reply:
still don't understand, care to give me an example?

---
In reply to: [230638268](https://github.com/Microsoft/pai/pull/1656#discussion_r230638268) [](ancestors = 230638268)

Reply:
I use the horovod json file as the example:
```
{
  "jobName": "horovod-mpi-cifar10",
  "image": "openpai/example.horovod.mpi",
  "dataDir": "$PAI_DEFAULT_FS_URI/examples/tensorflow/distributed-cifar-10/data",
  "outputDir": "$PAI_DEFAULT_FS_URI/examples/horovod/output",
  "codeDir": "$PAI_DEFAULT_FS_URI/examples/horovod/code",
  "virtualCluster": "default",
  "retryCount": 0,
  "taskRoles": [
    {
      "name": "main",
      "taskNumber": 1,
      "cpuNumber": 4,
      "memoryMB": 16384,
      "shmMB": 64,
      "gpuNumber": 2,
      "minFailedTaskCount": 1,
      "minSucceededTaskCount": 1,
      "command": "/bin/bash code/start.sh"
    },
    {
      "name": "worker",
      "taskNumber": 1,
      "cpuNumber": 4,
      "memoryMB": 16384,
      "shmMB": 64,
      "gpuNumber": 2,
      "minFailedTaskCount": 1,
      "command": "sleep infinity"
    }
  ]
}
```
To name one, if we remove the "minFailedTaskCount" key in next version of webportal, there will be an error when I use this json file to submit a horovod job. This will influence both users and the auto-test project.

Reply:
ok. you can just say if the format of the json is incorrect ...

---
In reply to: [230640443](https://github.com/Microsoft/pai/pull/1656#discussion_r230640443) [](ancestors = 230640443)

Reply:
fix